articles/tuples-data-structure-python/index.md

```diff
@@ -0,0 +1,307 @@
+## Tuple Data Structure in Python
+
+A tuple is an ordered collection of items. They are similar to lists but vary in the following aspects: They are immutable, (they cannot be changed) unlike lists which are mutable (they can be changed). Let us learn more about tuples and their related methods. We will also learn to effectively use them in Python.
```

Comment:
What do you mean by ordered? Is the order preserved?

Reply:
I gave an explanation for this in the latest commit, so that readers better understand :D